specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/ContainerAppsJob_Terminate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "containerAppsJobName": "testcontainerAppsJob0",
+    "api-version": "2022-11-01-preview"
+  },
+  "responses": {
+    "202": {}
```

Comment:
Same. How can user confirm the operation is completed

Reply:
With the job name in the path parameter with a status call